vendors/patterns.yml

```diff
@@ -41,4 +41,42 @@ patterns:
       - name: sendgrid.py
         start_offset: 20
         end_offset: 89
+
+  - name: Okta token
+    type: okta_token
+    regex:
+      version: 0.1
+      pattern: |
+        (0{2}[0-9A-Za-z\-\_]{40})
+      start: |
+        (\A|[^0-9A-Za-z\_\-0x])
```

Comment:
That 0x at the end won't do what you wanted. Take that out pls and add some extras to the "all hex" pattern (see the comment there)